### ISSUE TITLE
Use BUILD.bazel in the start script.

### ISSUE THREAD

**Issue by philderbeast** [tweag/rules_haskell#786]
Please use `BUILD.bazel` in the start script, see #412.

https://github.com/tweag/rules_haskell/blob/361793e3cf841545ef65ee242c25181808d623c5/start#L82

**Comment by philderbeast** [comment]
I have an existing `build` directory where I keep a [shake](http://hackage.haskell.org/package/shake) build project and got this error:

```
> ./rules-haskell-start.sh
./rules-haskell-start.sh: line 76: BUILD: Is a directory
```

Changing the start script to write to `BUILD.bazel` avoids this problem:

```
> ./rules-haskell-start.sh
WORKSPACE and initial BUILD files created. To run Bazel:

    $ bazel build //...
```

**Comment by mboes** [comment]
Fixed in #787 .
